README.md

```diff
@@ -29,7 +29,10 @@ rdhs::set_rdhs_config(
 
 6.  Create the folder `nmrSpaceTime/Data/{country}`
 
-7.  Create Admin 1 urban fraction file (`frame_urb_prop.csv` with columns "region" and "urban_prop"; following details here: <https://github.com/wu-thomas/SUMMER-DHS/tree/main/Vignette/Urban%20Fraction>) and put it in the country data folder.
+7.  Create the folder `nmrSpaceTime/Results/{country}`
```

Comment:
This is here: https://github.com/krpaulson/nmrSpaceTime/blob/main/0b_prep_UR.R#L520